### PR TITLE
Number of properties per population category

### PR DESCRIPTION
**Pull request by ketankartoza**
This Pr is for #664 

Before:
Screenshot 2023-08-03 at 2 55 59 PM
After:
Screenshot 2023-08-03 at 2 56 31 PM

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 12 min

File: django_project/frontend/utils/metrics.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+from collections import Counter
+
+from django.db.models import Sum
+from property.models import Property
+
+
+def calculate_population_categories(queryset):
```
Comment (dimasciput) [inline django_project/frontend/utils/metrics.py:7]: Hi @ketankartoza, could you please add a docstring here? It's a bit confusing to understand what the queryset is. Also, could you provide a type hint for the queryset?

File: django_project/frontend/utils/metrics.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    POPULATION_CATEGORY_DICT = {
```
Comment (dimasciput) [inline django_project/frontend/utils/metrics.py:8]: In Python convention, uppercase names are used for constants, which don't change once assigned.

File: django_project/frontend/api_views/metrics.py [hunk 3]
```diff
@@ -113,3 +114,19 @@ def get(self, request, *args, **kwargs):
             queryset, many=True, context={"request": request}
         )
         return Response(serializer.data)
+
+
+class PropertiesPerPopulationCategoryAPIView(APIView):
```
Comment (dimasciput) [inline django_project/frontend/api_views/metrics.py:119]: Please add docstring here too

**Review by dimasciput — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 1 min 22 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 15 min

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 14 min

**Pull request merged**